src/pages/Delivery/index.jsx

```diff
@@ -80,6 +80,7 @@ const Delivery = () => {
     >
       {center[0] && <Map center={center} dasher={dasher} permission={permission} />}
     </Layout>
+    </Fragment>
```

Comment:
Check identation. Please try to run linter

Reply:
done